hack/generated/controllers/crd_disk_test.go

```diff
@@ -48,10 +48,10 @@ func Test_Disk_CRUD(t *testing.T) {
 	armId := *disk.Status.Id
 
 	// Perform a simple patch.
-	patcher := tc.NewResourcePatcher(disk)
+	old := disk.DeepCopy()
 	networkAccessPolicy := compute.DiskPropertiesNetworkAccessPolicyDenyAll
 	disk.Spec.NetworkAccessPolicy = &networkAccessPolicy
-	patcher.Patch(disk)
+	tc.Patch(old, disk)
```

Comment:
Should this comment/code still be using the term _Patch_?

Reply:
This really is a Patch, so yes I think so.

If we wanted to use update here we'd need to also add an additional GET up above (so that we were operating on the latest version of the object). I figured keeping the Patch helper for tests was fine as it's a bit more succinct.

Reply:
Ok.